src/corner_kick/src/containers/Canvas/utils/draw.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * This file specifies functions to draw various types of shapes
+ */
```

Comment:
Is this for drawing shapes for spritesheet generation? If so we should def. make that explicit here.

Reply:
Done

Reply:
```
/***
 * This file specifies functions to draw various types of shapes. These functions
 * are used for spritesheet generation.
 */
```